test/test_cuda/requirements.txt

```diff
@@ -0,0 +1,18 @@
+accelerate
+autoawq
+auto-gptq
+datasets
+einops
+gptqmodel>=2.0
+intel-extension-for-pytorch==2.6.0
+intel-extension-for-transformers
+lm-eval>=0.4.2,<0.5
+numpy < 2.0
+optimum
+pandas
+pillow
+py-cpuinfo
+torch<2.7.0
```

Comment:
We shouldn't restrict the PyTorch version, as it may prevent us from identifying issues introduced in the latest release. For ITREX test, we can use a specific test to ensure compatibility.